.github/workflows/deploy-frontend-pr-previews.yml

```
@@ -598,6 +604,99 @@
+        uses: "google-github-actions/setup-gcloud@v1"
+
+      - name: Create google_application_credentials.json file
+        id: create-google-application-credentials-file
+        uses: jsdaniell/create-json@v1.2.2
+        with:
+          name: "google_application_credentials.json"
+          json: ${{ secrets.GCP_FRONTEND_PROJECT_SA_CREDENTIALS }}
+          dir: "website2/"
+
+      - name: Create app.yaml and .env.yaml files
+        run: |
```

actionlint

shellcheck reported issue in this script: SC2129:style:4:1: Consider using { cmd1; cmd2; } >> file instead of individual redirects

(shellcheck)